internal/e2e/testdata/graphql_type_User.txt

```diff
@@ -1,41 +1,41 @@
 queryString: graphql type User
 query: (and substr:"graphql" substr:"type" case_substr:"User")
-targetRank: 2
+targetRank: 21
 
-github.com/sourcegraph/sourcegraph/cmd/frontend/graphqlbackend/apitest/types.go
-47:type User struct {
-9:	Typename    string `json:"__typename"`
-32:	Typename    string `json:"__typename"`
-hidden 11 more line matches
-
-**github.com/sourcegraph/sourcegraph/cmd/frontend/graphqlbackend/schema.graphql**
```

Comment:
Can you explain why this does so much worse? It feels like it should be competitive, given it matches the path, plus `User`.